sklearn/datasets/twenty_newsgroups.py

```diff
@@ -216,11 +216,18 @@ def fetch_20newsgroups(data_home=None, subset='train', categories=None,
 
     Returns
     -------
-    bunch : Bunch object
-        bunch.data: list, length [n_samples]
-        bunch.target: array, shape [n_samples]
-        bunch.filenames: list, length [n_classes]
-        bunch.DESCR: a description of the dataset.
+    bunch : Bunch object with the following attribute:
+
+        - bunch.data: list, length [n_samples]
```

Comment:
I think you may not need these newlines in between these items.

Reply:
I remove it.